blocks/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+
+# Register your models here.
+
+from .models import Block
```

Comment:
- Use complete paths instead of relative
- Make sure to follow pep8 conventions on where to add new lines.